narwhal/types/proto/narwhal.proto

```diff
@@ -6,7 +6,7 @@ syntax = "proto3";
 package narwhal;
 
 message Transaction {
-    bytes transaction = 1;
+    repeated bytes transactions = 1;
```

Comment:
When we upgrade some validators with this new binary, would they start sending around batches in the new format? Would that cause older binaries to crash?

Reply:
Fortunately this is not used for Sui validator communications now. This is leftover from when Narwhal runs as an external consensus process for other systems. Since we are deprecating Narwhal in a few months, we have not prioritized cleaning up these interfaces.